src/management/Akka.Management/Dsl/AkkaManagement.cs

```diff
@@ -179,10 +179,14 @@ private ManagementRouteProviderSettings ProviderSettings()
             const string protocol = "http"; // changed to "https" if ManagementRouteProviderSettings.withHttpsConnectionContext is use
 
             var basePath = !string.IsNullOrWhiteSpace(Settings.Http.BasePath) ? Settings.Http.BasePath + "/" : string.Empty;
-            var selfBaseUri = new Uri($"{protocol}://{Settings.Http.Hostname}:{Settings.Http.Port}{basePath}");
+            var hostName = IsIPv6(Settings.Http.Hostname) ? $"[{Settings.Http.Hostname}]" : Settings.Http.Hostname;
```

Comment:
IPAddress.ToString() returns "2001:db8:85a3:8d3:1319:8a2e:370:7348" instead of "[2001:db8:85a3:8d3:1319:8a2e:370:7348]". This can create problems when you string interpolate it into an URI.